frontend/src/components/VerticalWord/CrosswordBoxContainer.tsx

```diff
@@ -36,8 +31,17 @@ const CrosswordBoxContainer = ({ crossword }: CrosswordBoxContainerProps) => {
     var columnIndex = cellNumber % dimension
     var rowIndex = Math.floor(cellNumber/dimension)
 
-    var input : string | undefined = event?.currentTarget?.textContent?.at(0)
-    grid[columnIndex][rowIndex] = input ? input : "";
+    var input : string | undefined = event?.currentTarget?.textContent?.at(0);

```

Comment:
please use let or const here, var is pretty much deprecated. 

var initializes this variable into the global scope and then gets assigned its value when it reaches its execution point. Its funky and weird and leads to worsened global variable pollution.[see this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#description)

Reply:
you can't blame me for javascript being shit hahahahahaha will change to let for all these